.github/workflows/ci.yml

```diff
@@ -59,14 +61,27 @@ jobs:
               run: echo "::add-matcher::${{ runner.tool_cache }}/phpunit.json"
 
             - name: Execute tests (Unit and Feature)
-              run: vendor/bin/phpunit
+              run: vendor/bin/phpunit --log-junit=reports/report-phpunit.xml --coverage-clover=reports/coverage-phpunit.xml
 
             - name: Static analysis with PHPStan
-              run: vendor/bin/phpstan analyse
+              run: vendor/bin/phpstan analyse --error-format=json > reports/phpstan.json
```

Comment:
I would be nice to keep the github phpstan report and also have the json report.
Please check this out.

Reply:
Which report are you referring to? I don't see a comment in any other PR with a phpstan report.